### ISSUE TITLE
Why dividing by normalized coordinates ?

### ISSUE THREAD

**Issue by anshkumar** [feiyuhuahuo/Yolact_minimal#15]
I was going through your code and I couldn't understand why are you dividing the mask_loss by area which is in normalized form ([code](https://github.com/feiyuhuahuo/Yolact_minimal/blob/9299a0cf346e455d672fadd796ac748871ba85e4/modules/multi_loss.py#L96)).  

The code is:
`mask_loss = mask_loss.sum(dim=(0, 1)) / pos_get_csize[:, 2] / pos_get_csize[:, 3]`

I guess you are trying to find the area of each prior box and divide it to the mask_loss. But the coordinates are in normalized from (if I'm not wrong).

**Comment by feiyuhuahuo** [comment]
You can see from the previous line code that `pos_get_csize` represents (center_x, center_y, width, height) of a prior box, so `/ pos_get_csize[:, 2] / pos_get_csize[:, 3]` means the dividing the area of  the prior box.

**Comment by anshkumar** [comment]
But `width` and `height` are still in normalized form. Why are we dividing by area in relative form ?

**Comment by feiyuhuahuo** [comment]
You mean  `width` and `height` are between 0~1? That doesn't matter. Because prior areas are used to balance loss of each mask. This avoid that if one mask has large area, it will have a large loss contribution. But certainly it will influence the final total mask loss because we use values between 0~1 instead of values in image scale (e.g. 300, 400). So we need a factor to multiply mask loss to balance the final mask loss, box loss, confidence loss and semantic loss.
